internal/rollup/rollup_bundle.bzl

```diff
@@ -441,12 +441,25 @@ def _rollup_bundle(ctx):
         _generate_code_split_entry(ctx, ctx.label.name + "_chunks", ctx.outputs.build_umd)
         _generate_code_split_entry(ctx, ctx.label.name + "_chunks", ctx.outputs.build_cjs)
 
-        # There is no source map explorer output when code-splitting but we still need to satisfy the output
-        ctx.actions.expand_template(
-            output = ctx.outputs.explore_html,
-            template = ctx.file._no_explore_html,
-            substitutions = {},
-        )
+        # There is no source map or explorer output when code-splitting but we still need to satisfy the output
```

Comment:
Seems like there's no source maps when code-splitting today? I guess that could be a future task...

Reply:
what does the rollup program produce? I bet it still creates sourcemaps, and having a size explorer for chunks is a pretty critical feature, could fix this next